docs/account-project-management/accounts-and-access/accounts-and-access-faqs.md

```diff
@@ -1,25 +1,25 @@
 
 If your account is inactive, please submit a reactivation request here: [https://my.alcf.anl.gov/accounts/#/accountReactivate](https://my.alcf.anl.gov/accounts/#/accountReactivate).
 
 If you never had an ALCF account, please apply for one here: [https://my.alcf.anl.gov/accounts/#/accountRequest](https://my.alcf.anl.gov/accounts/#/accountRequest). Note that all ALCF accounts must be associated with a project with an active allocation.
 
 ## How do I request a new project/allocation?
-There are 3 allocation opportunities at ALCF. Please see How to Get an Allocation on how to get time on our systems.
+There are three allocation opportunities at ALCF. Please see "How to Get an Allocation" on how to get time on our systems.
 
 ## Who do I contact if my Discretionary Project Allocation expires or if I need to request additional hours?
-To request an extension of your existing discretionary allocation or to request additional hours, please email support@alcf.anl.gov with answers to the following or fill out the form at request an extension/additional hours:
-- What you have accomplished with your original allocation?
+To request an extension of your existing discretionary allocation or to request additional hours, please email support@alcf.anl.gov with answers to the following or fill out the form to request an extension/additional hours:
+- What have you accomplished with your original allocation?
   - Please include a brief description of any publications or major presentations that were (or will be) generated in full or in part because of this allocation.
-- What you will do with the extra time?
-- What you are requesting as your new expiration date?
-- How many additional hours you are requesting?
+- What will you do with the extra time?
+- What are you requesting as your new expiration date?
+- How many additional hours are you requesting?
 
 ## How do I join a project?
-To join a project, please go to [https://my.alcf.anl.gov/](https://my.alcf.anl.gov/), then click "join a project". Once there, scroll down to the project you want to join and click on it. At the bottom of the next page, please click on the "Request Membership" button. Once we receive approval from the PI regarding your membership request, we will provide you with access to the necessary resources.
+To join a project, please go to [https://my.alcf.anl.gov/](https://my.alcf.anl.gov/), then click "join a project." Once there, scroll down to the project you want to join and click on it. At the bottom of the next page, please click on the "Request Membership" button. Once we receive approval from the PI regarding your membership request, we will provide you with access to the necessary resources.
```

Comment:
this description is out-of-date. Only the URL was updated when MyALCF was launched